src/mgos_gcp.c

```diff
@@ -66,10 +72,23 @@ static void base64url_putc(char c, void *arg) {
   mbuf_append(&ctx->jwt, &c, 1);
 }
 
+static void mgos_gcp_jwt_timeout(void *param) {
+  struct gcp_state *state = (struct gcp_state *)param;
+  state->jwt_timeout = 0;
+  LOG(LL_DEBUG, ("Disconnecting global MQTT connection due to JWT timeout."));
```

Comment:
make it INFO level and change wording somewhat: "Dropping MQTT connection due to imminent token expiration"